azavea/cac-tripplanner
# Feature/kak/fix ansible provisioning#1213

## Overview

Completes removal of python 2 usage. Inlines the PostgreSQL support role installs to work around the ansible role's dependency on python 2.

## Notes

I did this while working on the ansible role updates for #1213. So to provision, it will be necessary to either use ansible < 2.9, or copy over the `azavea.nginx` role from azavea/ansible-nginx#11 and the `azavea.postgresql` from azavea/ansible-postgresql#22.


## Testing Instructions

 * check python 2 is no longer in use, either directly or by dependent role
 * ensure a built graph (or recent files to build a graph) are in `otp_data`
 * `vagrant destroy`
 * `cd src && rm -r node_modules && npm i` (reinstalling on host first prevents file sync issues)
 * `vagrant up`

Fixes #1151
